src/pages/diagnose/healthchecks.md

```diff
@@ -14,4 +14,18 @@ First, make sure your webserver has an endpoint (e.g. `/health`) that will retur
 
 Under Service → Settings, input your health endpoint. Railway will wait for this endpoint to serve a 200 status code before switching traffic to your new deployment.
 
+Healthchecks run instantly and add no overhead time to the deployment process.
```

Comment:
Not technically true would remove

Reply:
This was the copy that was added in Healthchecks 2.0, will remove it.